.cypress/integration/7_app_analytics.spec.js

```diff
@@ -389,12 +394,13 @@ describe('Viewing application', () => {
     cy.get('[data-test-subj="superDatePickerApplyTimeButton"]').click();
     cy.wait(delay);
     cy.get('[data-test-subj="main-content-visTab"]').click();
-    suppressResizeObserverIssue();
     cy.get('.euiTab[id="availability-panel"]').click();
-    cy.get('[title="Bar"]').click();
+    cy.wait(delay);
+    cy.get('[data-test-subj="comboBoxInput"]').click();
+    cy.focused().type('{downArrow}');
```

Comment:
duplicate to line below?

Reply:
Was used to select TimeSeries as the third option by going down twice, adding a commit to make this more clear and select it in a better way. It previously selected Horizontal Bar which did not have the availability function.